packages/admin-panel-server/src/routes/SaveDashboardVisualisationRoute.ts

```diff
@@ -50,12 +52,12 @@ export class SaveDashboardVisualisationRoute extends Route {
     let result;
 
     // Update visualisation if id exists
-    if (visualisation.id) {
-      result = await this.meditrakConnection.updateResource(`dashboardVisualisations/${visualisation.id}`, {}, body);
+    if (dashboardVisualisationId) {
```

Comment:
I've changed the save route (save an existing dashboard viz) to `/PUT dashboardVisualisation/:dashboardVisualisationId` to make it a bit more resty. However, Create/Save dashboard viz still share the same RouteHandler

Reply:
Nice one. Resty is good.